src/features/smartSearch/components/filters/CallHistory/index.tsx

```diff
@@ -165,7 +165,19 @@ const CallHistory = ({
                 ))}
               </StyledSelect>
             ),
-            minTimes: filter.config.minTimes ?? 1,
+            minTimes:
+              filter.config.minTimes === 1 ||
+              filter.config.minTimes === undefined ? (
+                <></>
```

Comment:
I think this conditional is unnecessary. The word "time" or "times" need to be visible in edit mode. Right now it says "Add people who have been called at least 1 in any assignment", which is not a correct sentence.

<img width="1178" alt="image" src="https://github.com/zetkin/app.zetkin.org/assets/550212/907825e6-731d-4ecb-9603-28b115f40661">